app/views/markets/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Markets</h1>
+<% if @market_link == true %>
```

Comment:
Since you are checked for equivalence to `true`, you don't really need to add the extra `== true` since that is what the conditional is checking for already